packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -350,7 +352,40 @@ export default class QuanticCaseClassification extends LightningElement {
    * @returns {void}
    */
   setFieldValue(value) {
-    this.field.update(value);
+    if (this.field.state.value !== value) {
+      this.field.update(value);
+    }
     this._value = value;
   }
+
+  /**
+   * Indicates whether a value is one of the proposed suggestions.
+   * @param {string} value
+   * @returns {boolean}
+   */
+  isSuggestion(value) {
+    return this.suggestions.find((suggestion) => suggestion.value === value);
```

Comment:
`.find()` returns the array element if found. If we're just looking for a boolean value using `.includes()` may be more appropriate.

Reply:
[Array.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) only works with value types. In this case [Array.some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) should be used.

Reply:
Fixed👌